### PR TITLE
[OSDEV-1100] Use the new production location naming in the Logstash app.

### PR DESCRIPTION
**Pull request by vladsha-dev**
[OSDEV-1100](https://opensupplyhub.atlassian.net/browse/OSDEV-1100)

- Replaced all mentions of "facility" and "facilities" with the new production location naming in the Logstash app.
- Renamed `location` field in the production locations index to `coordinates`.

### CODE REVIEW

File: src/logstash/indexes/production_locations.json
Comment (vladsha-dev) [inline src/logstash/indexes/production_locations.json:1]: I tried to force Git to show with "git mv" that this file was renamed, moved, and changed a little bit, but it still shows that it is newly created.

File: src/logstash/scripts/facilities/facility_type.rb
Comment (vladsha-dev) [inline src/logstash/scripts/facilities/facility_type.rb:1]: I tried to force Git to show with "git mv" that this file was renamed, moved, and changed a little bit, but it still shows that it is newly created.

File: src/logstash/scripts/production_locations/location_type.rb
Comment (vladsha-dev) [inline src/logstash/scripts/production_locations/location_type.rb:1]: I tried to force Git to show with "git mv" that this file was renamed, moved, and changed a little bit, but it still shows that it is newly created.
It is former `facility_type.rb` file.

**Review by VadimKovalenkoSNF — DISMISSED** [review]
Looks ok. Approved.

**Review by roman-stolar — APPROVED** [review]
LGTM

**Review by VadimKovalenkoSNF — APPROVED** [review]
LGTM

**Review by roman-stolar — APPROVED** [review]

**Pull request merged**